### PR TITLE
docs: update user guide for the ask-cli getting started section

### PR DESCRIPTION
**Pull request by RonWang**
## Description
Update user guide for the ask-cli getting started section to correctly reflect the user flow. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Docs(Add new document content)
- [ ] Translate Docs(Translate document content)

## Checklist

- [x] My code follows the code style of this project
- [x] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [x] I have read the **README** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

[issues]: https://https://github.com/alexa/ask-sdk-controls/issues
[license]: http://aws.amazon.com/apache2.0/
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

**Review by Shreyas-vgr — APPROVED** [review]

File: doc/userGuide.md [hunk 1]
```diff
@@ -175,9 +175,9 @@ for more information.
 
 ```bash
 ask new
-> choose a skill hosting method.
 > select NodeJS
-> select Hello World (with Controls)
+> choose a skill hosting method, except "Alexa-hosted skills" as it doesn't support custom template selection
```
Comment (nikhilym) [inline doc/userGuide.md:179]: You will also need to make changes in the paragraph just before this snippet. Also, I think it is better to mention that `Alexa-hosted skills` custom template selection is not supported at the moment?

Comment (RonWang) [reply doc/userGuide.md]: Good call, done

File: doc/userGuide.md [hunk 2]
```diff
@@ -162,21 +162,18 @@ guide](https://developer.amazon.com/en-US/docs/alexa/smapi/ask-cli-v1-to-v2-migr
...
 deploy your skill to test a skill end-to-end with a device - the code can simply run on
 your local machine and attached to the VScode debugger. If you are unsure which hosting
-method to choose, we recommend 'Alexa-hosted' which is free and easy to use. See [Creating
+method to choose, please see [Creating
 new skill with
-ASK-CLI](https://developer.amazon.com/en-US/docs/alexa/smapi/ask-cli-intro.html#create-new-skill)
-and [Building an Alexa-hosted
-skill](https://developer.amazon.com/en-US/docs/alexa/hosted-skills/build-a-skill-end-to-end-using-an-alexa-hosted-skill.html)
-for more information.
+ASK-CLI](https://developer.amazon.com/en-US/docs/alexa/smapi/ask-cli-intro.html#create-new-skill) for more information. Please note `Alexa-hosted skills` is not supported as an option at the moment, because custom template selection is available.
```
Comment (nikhilym) [inline doc/userGuide.md:171]: is it a typo : "because custom template selection is **not** available"?

**Review by nikhilym — APPROVED** [review]
:shipit:

**Review by mikelid — APPROVED** [review]

**Pull request merged**